docs/app/references/migration-guide.mdx

```diff
@@ -20,6 +20,14 @@ Cypress requires [Node.js](https://nodejs.org/en) in order to install the Cypres
 Node.js versions 18 and 23 are no longer supported.
 [See Node's release schedule](https://github.com/nodejs/Release).
 
+### Unsupported Linux Distributions
+
+Prebuilt binaries for Linux are no longer compatible with Linux distributions based on glibc `<2.31`.
+This support is in line with Node.js's support for Linux in 20+.
+
+If you're using a Linux distribution based on glibc `<2.31`, for example, RHEL 8, CentOS 8, AlmaLinux 8, Rocky Linux 8, you'll need to
+update your system to a newer version to install Cypress 15+.
```

Comment:
```suggestion
If you're using a Linux distribution based on glibc `<2.31`, you'll need to
update your system to a newer version to install Cypress 15+.
To display which version of glibc your Linux system is running, execute `ldd --version`.
```

This avoids a partial enumeration of affected Linux distros, which could imply support. These distros were however not previously mentioned. CentOS 8 already reached EOL in 2021. The above change is safer and more general.